persistence/aggregate.go

```diff
@@ -32,6 +35,20 @@ type AggregateMetaData struct {
 	BarrierEventID string
 }
 
+type AggregateSnapshot struct {
+	// HandlerKey is the identity key of the aggregate message handler.
+	HandlerKey string
+
+	// InstanceID is the aggregate instance ID.
+	InstanceID string
+
+	// Version is the instance's current version.
+	Version string
```

Comment:
```suggestion
	// LastEventID is the most recent event ID applied to the aggregate root at the point this snapshot was taken.
	LastEventID string
```

Suggestion for clarity.